lib/MTT/Test/Build/Trivial.pm

```diff
@@ -56,6 +58,12 @@ sub Build {
                        "trivial_tests_cflags");
     my $fflags = Value($ini, $config->{full_section_name}, 
                        "trivial_tests_fflags");
+    my $f77flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f77flags");
+    my $f90flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f90flags");
+    my $f08flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f08flags");
```

Comment:
I'm sorry -- these names are wrong.  They are not the "F77 bindings", "F90 bindings", and "F08 bindings", they are the "mpif.h interface", the "'use mpi' interface", and the "'use mpi_f08' interface".

For example: the mpif.h interface does not conform to the Fortran 77 standard.  Nor does the "use mpi" interface conform to Fortran 90.